src/effects/deeplearning/ModelManagerPanel.cpp

```diff
@@ -0,0 +1,809 @@
+
+      // model size
+      mModelSize = S.AddVariableText(XO("[- MB]"));
+      FetchModelSize();
+
+   }
+   S.EndMultiColumn();
+   // S.EndHorizontalLay();
+
+   // by {author}
+   S.StartHorizontalLay(wxALIGN_LEFT, true);
+   {
+      S.AddVariableText(XO("by"));
+      mModelAuthor = S.AddVariableText(XO("%s")
+                                              .Format(mCard->author()));
+      mModelAuthor->SetFont(wxFont(wxFontInfo().Bold()));
+   }
+   S.EndHorizontalLay();
+}
+
+void ModelCardPanel::PopulateDomainTags(ShuttleGui &S)
+{
+   S.StartHorizontalLay(wxALIGN_LEFT | wxALIGN_TOP, true);
+   {
+      for (auto &tag : mCard->domain_tags())
+      {
+         S.AddWindow(
+            safenew DomainTagPanel(this, tag, mTagColors[tag])
+         );
+      }
+   }
+   S.EndHorizontalLay();
+}
+
+void ModelCardPanel::PopulateShortDescription(ShuttleGui &S)
+{
+   // model description
+   // S.StartStatic(XO("Description"));
+   S.SetBorder(10);
+   mShortDescription = S.AddVariableText(
+                                       XO("%s").Format(wxString(mCard->short_description())),
+                                       false, wxLEFT);
+   S.SetBorder(10);
+   // S.EndStatic();
+}
+
+void ModelCardPanel::PopulateLongDescription(ShuttleGui &S)
+{
+   // model description
+   S.StartStatic(XO(""));
+   mLongDescription = S.AddVariableText(
+                                       XO("%s").Format(wxString(mCard->long_description())),
+                                       false, wxLEFT, 
+                                       GetSize().GetWidth() - 30);
+   S.EndStatic();
+}
+
+void ModelCardPanel::PopulateMetadata(ShuttleGui &S)
+{
+   S.StartMultiColumn(2, wxALIGN_LEFT);
+   {
+      S.AddVariableText(XO("Effect: "))
+          ->SetFont(wxFont(wxFontInfo().Bold()));
+      S.AddVariableText(XO("%s")
+                            .Format(mCard->effect_type()));
+
+      S.AddVariableText(XO("Sample Rate: "))
+          ->SetFont(wxFont(wxFontInfo().Bold()));
+      S.AddVariableText(XO("%d")
+                            .Format(mCard->sample_rate()));
+
+      std::string tagString;
+      for (auto &tag : mCard->tags())
+      {
+         if (!tagString.empty())
+            tagString = tagString +  ", " + tag;
+         else
+            tagString = tag;
+      }
+      
+      S.AddVariableText(XO("Tags: "))
+          ->SetFont(wxFont(wxFontInfo().Bold()));
+      S.AddVariableText(XO("%s")
+                            .Format(tagString));
+   }
+   S.EndMultiColumn();
+}
+
+void ModelCardPanel::PopulateInstallCtrls(ShuttleGui &S)
+{
+   DeepModelManager &manager = DeepModelManager::Get();
+
+   S.StartVerticalLay(wxCENTER, true);
+   {
+      // add a progress gauge for downloads, but hide it
+      mInstallProgressGauge = safenew wxGauge(S.GetParent(), wxID_ANY, 100); // TODO:  sizing
+      mInstallProgressGauge->SetSize(wxSize(80, 20));
+      S.AddWindow(mInstallProgressGauge);
+      mInstallProgressGauge->Show();
```

Comment:
Why do we need Show?

Reply:
https://github.com/audacity/audacity/pull/1384/commits/e3a958eabcae9c70e5c3c94bfab6993f2ccef402